src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
```

Comment:
You use Shape values in several places in your code, so it makes sense to make it `Enum`, not just `type`.
Use `Enum` here instead